lib/provider/aws/clean-up-event.js

```diff
@@ -12,13 +12,55 @@ function removeBasePath(path = '/', basePath) {
   return path;
 }
 
+function recursiveURLDecode(value) {
+
+  if (typeof value === 'string' || value instanceof String) {
```

Comment:
more a nitpick or curiosity, is there a situation where value typeof value is not a string, but it is an instaceof string?

Reply:
It's a weird corner case that would probably never come up in this instance, but check out the following code:

```
let x = new String("blah");
console.log(typeof x);
```
`"object"`

I don't see many strings created that way, so it's unlikely to be an issue, but it's still technically possible.

Reply:
oh geez, forgot about that, thanks :)